src/trade-params/FullTradeParams.js

```diff
@@ -456,6 +456,10 @@ export default class FullTradeParams extends Component {
                     shown={!!trade.proposalError}
                     text={trade.proposalError ? trade.proposalError.message.split(')')[1] : ''}
                 />
+                <ErrorMsg
+                    shown={!!trade.contractForError}
```

Comment:
Sometimes I feel it's more obvious that I want it to be boolean though ~~
Please forgive my paranoid XD